templates/d/buildcart.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+rm -f game.tic
+make clean
+make
+tic80 --fs . --cmd 'load wasmdemo.wasmp & import binary cart.wasm & save game.tic & exit'
+tic80 --fs . --cmd 'load game.tic & run & exit'
```

Comment:
Why do we need to invoke tic80 twice vs doing it all in a single session?

Reply:
My intention is to demonstrate the two separate actions -- creating a .tic file from necessary artifacts, and running the .tic directly without reference to said artifacts -- for the programmer who is new to this console.